src/Google.Cloud.Functions.Framework/LegacyEvents/Context.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2020, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Google.Cloud.Functions.Framework.LegacyEvents
+{
+    /// <summary>
+    /// The context of an event.
+    /// </summary>
+    public class Context
```

Comment:
nit: This name is misleading? Maybe EventDescripto?

Reply:
While I generally agree, this is the term that's generally used in documentation etc.
(The class should be sealed though - will fix that!)